wix.json

```diff
@@ -0,0 +1,27 @@
+{
+  "product": "torus",
+  "company": "manifoldco",
+  "license": "LICENSE.md",
+  "version": "0.0.1",
```

Comment:
This is the version of the binary right? Ideally this file can be tempalted like `package.json` inside `packaging/npm` which is then filled in prior to packaging.

This way we can pull the correct version in as well.